SDPi_Supplement/asciidoc/volume2/dev-28/tf2-ch-a-mdpws-dev-28.adoc

```diff
@@ -0,0 +1,28 @@
+:var_uri_ws_eventing_notification: https://www.w3.org/Submission/2006/SUBM-WS-Eventing-20060315/#Notifications
+
+[#vol2_clause_appendix_mdpws_dev_28]
+==== MDPWS: Notify Change in System Context and Capabilities  [DEV-28]
+
+
+// ---------- EpisodicContextReport ---------
+
+
+===== {var_label_dev_28_message_contextreport} Message
+
+The <<vol2_clause_dev_28_message_notification, {var_label_dev_28_message_contextreport}>> message is encoded by using {var_uri_dpws_messaging}[DPWS Messaging].

```

Comment:
For some reason, the link `{var_uri_dpws_messaging}[DPWS Messaging]` is not rendered correctly. Maybe it makes sense to come up with a file that globally defines variables used throughout the document, esp. to prevent us from having this weird side effects. For now, I guess the solution to the problem is the redefinition of the link within the boundaries of this file.